src/Discord.Net.WebSocket/Entities/Interaction/Modals/SocketModal.cs

```diff
@@ -0,0 +1,295 @@
+using Discord.Net.Rest;
+using Discord.Rest;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using DataModel = Discord.API.ModalInteractionData;
+using ModelBase = Discord.API.Interaction;
+
+namespace Discord.WebSocket
+{
+    /// <summary>
+    ///     Represents a user submitted <see cref="Discord.Modal"/> received via GateWay.
```

Comment:
```suggestion
    ///     Represents a user submitted <see cref="Discord.Modal"/> received via Gateway.
```

Unsure of the capitalisation of the `w`?